wolfcrypt/src/sha512.c

```diff
@@ -895,13 +1022,40 @@ int wc_Sha512FinalRaw(wc_Sha512* sha512, byte* hash)
     return 0;
 }
 
-int wc_Sha512Final(wc_Sha512* sha512, byte* hash)
+static int Sha512_Family_Final(wc_Sha512* sha512, byte* hash, 
+                                                    enum wc_HashType type)
 {
     int ret;
+    int digest_sz;
+    int (*initfp)(wc_Sha512*);
+
+    (void)initfp;
 
     if (sha512 == NULL || hash == NULL) {
         return BAD_FUNC_ARG;
     }
+    
+    if (type == WC_HASH_TYPE_SHA512) {
+        initfp    = InitSha512;
+        digest_sz = WC_SHA512_DIGEST_SIZE;
+    }
+#if !defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)
+#if !defined(WOLFSSL_NOSHA512_224)
+    else if (type == WC_HASH_TYPE_SHA512_224) {
+        initfp    = InitSha512_224;
+        digest_sz = WC_SHA512_224_DIGEST_SIZE;
```

Comment:
I believe we prefer camelCase to snake_case. So `digest_sz` -> `digestSz`.

Reply:
Changed  snakes to camels.